backend/FwLite/LcmCrdt/QueryHelpers.cs

```diff
@@ -1,11 +1,14 @@
+using System.Globalization;
+
 namespace LcmCrdt;
 
 public static class QueryHelpers
 {
-    public static void ApplySortOrder(this Entry entry)
+    public static void ApplySortOrder(this Entry entry, CompareInfo sortCompareInfo)
     {
         entry.Senses.ApplySortOrder();
         entry.Components.ApplySortOrder();
+        entry.ComplexForms.Sort((a, b) => ComplexFormsComparison(a, b, sortCompareInfo));
```

Comment:
it's a minor thing, but since `ApplySortOrder` is getting called once for each entry I think it's worth trying to avoid an allocation here (due to capturing `sortCompareInfo` in the callback). I would change the signature of `ApplySortOrder` to take an `IComparable<ComplexFormComponent>`, this basically lets you move the allocation outside of the for each entry loop.